<launch>
  <node pkg="bt_sim_helpers" type="sim_pickups_node.py" name="sim_pickups" respawn="true">
    <param name="object_frame_prefix" type="str" value="object" />
    <param name="world_frame" type="str" value="map" />
    <param name="max_interact_distance" type="double" value="0.2" />
    <rosparam param="positions">
      [[9, 8], [10.5, 3]]
    </rosparam>
  </node>
  <node pkg="bt_sim_helpers" type="sim_doors_node.py" name="sim_doors" respawn="true">
    <param name="door_frame_prefix" type="str" value="door" />
    <param name="world_frame" type="str" value="map" />
    <param name="max_interact_distance" type="double" value="1.0" />
    <rosparam param="positions">
      [[7.5, 4.5], [9.5, 4.5]]
    </rosparam>
  </node>
</launch>
